client/homebrew/editor/metadataEditor/metadataEditor.jsx

```diff
@@ -18,7 +18,8 @@ const MetadataEditor = createClass({
 				published   : false,
 				authors     : [],
 				systems     : [],
-				renderer    : 'legacy'
+				renderer    : 'legacy',
+				language   	: 'en'
```

Comment:
Run the lint script if you haven't and align the colons if it doesn't automatically. I think everything here is using spaces to align colons.